config/initializers/content_security_policy.rb

```diff
@@ -38,7 +38,7 @@
   config.content_security_policy_nonce_directives = %w[script-src style-src]
 
   # Report violations without enforcing the policy.
-  config.content_security_policy_report_only = true
+  config.content_security_policy_report_only = false
```

Comment:
Did you check for any violations in the logs? I fully expect there to be some we need to add in before we enable this in prod 😅

Reply:
Tks @ethax-ross I had not checked the browser logs... it's clear from issues now.